src/sentry/data_export/tasks.py

```diff
@@ -103,6 +103,11 @@ def assemble_download(
         try:
             current.retry()
         except MaxRetriesExceededError:
+            metrics.incr(
+                "dataexport.end",
+                tags={"success": False, "error": six.text_type(error)},
```

Comment:
It might be worth adding something here that lets you figure out which job failed. An ID, a query or whatever.

Reply:
that's already available in sentry via `capture_exception` a few lines up, do you think its worth having this redundant information?

Reply:
I tend to believe in more information is better. If you are looking at this metric in datadog, how do you debug it? Is it easy to connect to the sentry errors? Would adding something else here help save time?

If it's not useful then feel free to ignore.

Reply:
I believe the workflow here is to see there is an error and jump to sentry as there simply isn't enough useful information in datadog beyond what type of error occurred and its frequency to debug it. It's also not trivial to find the error in sentry by the id compared to the error message.